art12/models.py

```diff
@@ -458,3 +464,53 @@ def upgrade(revision='head'):
 @db_manager.command
 def downgrade(revision):
     return alembic(['downgrade', revision])
+
+
+@db_manager.command
+def json_dump(model):
+    thismodule = sys.modules[__name__]
+    base_class = getattr(thismodule, model)
+
+    entries = base_class.query.all()
+    relationship_fields = [rfield for rfield, _ in inspect(base_class).relationships.items()]
+    model_fields = [field for field in inspect(base_class).attrs.keys() if field not in relationship_fields]
+
+    objects = []
+    primary_keys = []
+
+    for entry in entries:
+        kwargs = {
+            "model": model,
+            "filter_fields": ",".join(primary_keys),
```

Comment:
primary_keys are null
You can find if a field is primary key here: inspect(base_class).attrs.dataset_id.columns[0].primary_key